extensions/warp/src/launch-config.tsx

```diff
@@ -118,6 +176,20 @@ function SearchListItem({ searchResult }: { searchResult: SearchResult }) {
               title="Save as Quicklink"
               quicklink={{ link: launchConfig(searchResult.name), name: searchResult.name }}
             />
+            {!isSearching && (
+              <>
+                <Action
+                  title="Move up"
+                  shortcut={{ modifiers: ["cmd", "shift"], key: "arrowUp" }}
+                  onAction={moveSearchResultUp}
+                />
+                <Action
+                  title="Move down"
+                  shortcut={{ modifiers: ["cmd", "shift"], key: "arrowDown" }}
+                  onAction={moveSearchResultDown}
+                />
+              </>
```

Comment:
Missing icons 🙂

Reply:
Is there any specific convention I should follow when selecting the icons? I didn't find information on what icon I should choose for such generic operations such as MoveUp/MoveDown. 

![Screenshot 2024-08-06 at 13 37 56](https://github.com/user-attachments/assets/81454c73-94cb-45bb-af68-ec8064c1b950)

Reply:
It looks like you're using the same icons as we do in Snippets so it should be just fine 🙂

<img width="383" alt="image" src="https://github.com/user-attachments/assets/4a11d39b-d9dd-45ec-8ee6-fc65071443ff">

We also have a generic shortcut for moving which is used in Raycast, you can use `shortcut={Keyboard.Shortcut.Common.MoveDown}` (and Up)

Reply:
I had to migrate to newest version of the API since current version didn't have the `Keyboard.Shortcut.Common` implemented yet. 

I also swapped swapped couple of hardcoded shortcuts to use the `Keyboard.Shortcut.Common` instead.
I only did that in places where shortcuts where exactly the same. 

I found one place where the shortcuts seem to be wrong. It's inside the [open-directory.tsx](extensions/warp/src/open-directory.tsx) file on lines `201` and `210`. For some reason it uses (⌘⌥↑) and (⌘⌥↓) instead of (⌘⇧↑) and " (⌘⇧↓) which are the recommended shortcuts right? Let me know if I shall change it. 

![Screenshot 2024-08-06 at 16 12 30](https://github.com/user-attachments/assets/d5c485dc-81d2-4bd5-9d6e-98b644bd860e)

Reply:
I would change that to maintain consistency 🙂

Reply:
Changed 😃 I  also added the showToast operation to maintain consistency. I saw that such Toast is invoked when moving the pinned directories in warp extension or when moving pinned snippets in SearchSnippets command.